src/lecture_feedback/user_stats_tracker.py

```diff
@@ -37,7 +37,8 @@ def set_user_active(self, user_id: str) -> None:
             self._user_stats[user_id].last_seen = time.time()
 
     def get_user_stats(self) -> dict[str, UserData]:
-        return self._user_stats.copy()
+        # Return a plain dict view (shallow copy) of the current user stats
```

Comment:
how about renaming the function:
get_user_stats_copy then we could remove comment 
(https://www.youtube.com/watch?v=Bf7vDBBOBUA)